jfxtras-agenda/src/test/java/jfxtras/scene/control/agenda/test/AgendaRenderTest.java

```diff
@@ -153,22 +161,103 @@ public void renderRegularAppointmentCalendar()
+            );
+        });
+
+        Node n = find("#AppointmentWholedayBodyPane2014-01-01/0");
+        //AssertNode.generateSource("n", n, null, false, jfxtras.test.AssertNode.A.XYWH);
+        new AssertNode(n).assertXYWH(0.5, 0.0, 5.0, 1006.125, 0.01);
+        //TestUtil.sleep(3000);
+    }
+    
+    @Test
+    public void renderWholeDayAppointmentHijrahDate()
+    {
+        TestUtil.runThenWaitForPaintPulse( () -> {
+            Temporal h = HijrahDate.from(LocalDate.of(2014, 1, 1));
+            agenda.appointments().add( new Agenda.AppointmentImplTemporal()
+                .withStartTemporal(h)
+                .withEndTemporal(h.plus(1, ChronoUnit.DAYS))
+                .withWholeDay(true)
+                .withAppointmentGroup(appointmentGroupMap.get("group01"))
+            );
+        });
+
+        Node n = find("#AppointmentWholedayBodyPane2014-01-01/0");
+        //AssertNode.generateSource("n", n, null, false, jfxtras.test.AssertNode.A.XYWH);
+        new AssertNode(n).assertXYWH(0.5, 0.0, 5.0, 1006.125, 0.01);
+        //TestUtil.sleep(3000);
+    }
+    
+    @Test
+    public void renderRegularAppointmentOffset()
+    {
+        TestUtil.runThenWaitForPaintPulse( () -> {
+            ZoneOffset offset = ZonedDateTime.now().getOffset();
+            agenda.appointments().add( new Agenda.AppointmentImplTemporal()
+                .withStartTemporal(OffsetDateTime.of(LocalDateTime.of(2014, 1, 1, 10, 0), offset))
+                .withEndTemporal(OffsetDateTime.of(LocalDateTime.of(2014, 1, 1, 12, 0), offset))
+                .withAppointmentGroup(appointmentGroupMap.get("group01"))
+            );
+        });
+
+        Node n = find("#AppointmentRegularBodyPane2014-01-01/0");
+        //AssertNode.generateSource("n", n, null, false, jfxtras.test.AssertNode.A.XYWH);
+        new AssertNode(n).assertXYWH(0.5, 419.5, 125.0, 84.0, 0.01);
+        //TestUtil.sleep(3000);
+    }
+    
+    @Test
+    public void renderRegularAppointmentInstant()
+    {
+        TestUtil.runThenWaitForPaintPulse( () -> {
+            SimpleDateFormat sdf = new SimpleDateFormat("yyyy-M-dd hh:mm:ss");
+            String dateInString = "2014-1-1 10:00:00";
+            Date date = null;
+            try {
+                date = sdf.parse(dateInString);
+            } catch (Exception e) {
+                e.printStackTrace();
+            }
+            Instant instant = date.toInstant();
+            agenda.appointments().add( new Agenda.AppointmentImplTemporal()
+                .withStartTemporal(instant)
+                .withEndTemporal(instant.plus(2, ChronoUnit.HOURS))
+                .withAppointmentGroup(appointmentGroupMap.get("group01"))
+            );
+        });
+
+        Node n = find("#AppointmentRegularBodyPane2014-01-01/0");
+        //AssertNode.generateSource("n", n, null, false, jfxtras.test.AssertNode.A.XYWH);
+        new AssertNode(n).assertXYWH(0.5, 419.5, 125.0, 84.0, 0.01);
```

Comment:
I am still having different Y values - Y expected:<419.5> but was:<402.5>

I thought the Robo font was suppose to fix that, right?